monai/data/image_reader.py

```diff
@@ -294,21 +290,16 @@ def _get_array_data(self, img):
         The first axis of the returned array is the channel axis.
 
         Args:
-            img: a ITK image object loaded from a image file.
+            img: an ITK image object loaded from an image file.
 
         """
         channels = img.GetNumberOfComponentsPerPixel()
+        np_data = itk.array_view_from_image(img).T
```

Comment:
Hi @wyli ,

May I know why you use `.T` here?

Thanks.

Reply:
this is to be consistent with numpy/nibabel indexing convention

Reply:
I am developing the ITK writer, so should we also add `.T` in saving logic to be consistent?
https://github.com/Project-MONAI/MONAI/pull/2647/files#diff-330a471ba237244dca9eda33afe0b5ee8516c8986e97eb4761de9d52dafe2fd6R152

Thanks.